rskj-core/src/main/java/co/rsk/core/bc/MainchainView.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2019 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.core.bc;
+
+public interface MainchainView {
```

Comment:
Is this interface still needed? It's empty and no one uses it.

Reply:
It's a [Marker Interface](https://en.wikipedia.org/wiki/Marker_interface_pattern) intended to group all the views interfaces. All the views use it.